README.md

```diff
@@ -124,7 +122,7 @@ An ontology can be integrated into the graph. By doing so, it is possible to gai
 ***
 # Plugin requirements and installation
 
```

Comment:
probably major version compatibility is sufficient? assuming renku follows semver.

Reply:
I assume you meant to adapt the paragraph so to mention that a minimum version of renku is needed?
Or also the title?